kafe2/fit/tools/contours_profiler.py

```diff
@@ -163,9 +164,10 @@ def __init__(
         profile_subtract_min=True,
         profile_bound=2.45,
         contour_points=100,
-        contour_sigma_values=(1.0, 2.0),
+        contour_sigma_values=None,
```

Comment:
```suggestion
        contour_sigma_values=None,
        contour_cl_values=None,
```

I think the two arguments should be next to each other in the signature. In principle this would lead to breakage if someone is calling this constructor with only positional arguments but I don't expect that we have even a single user doing this.